index.html

```diff
@@ -45,5 +45,117 @@ <h2>LET'S CONNECT</h2>
       </ul>
       
     </section>
```

Comment:
- KIndly make sure your design look like the Figma design at the following points:

  - [ ] There is a gap between the headline section and the work section.
  - Your design:
![Screenshot from 2023-03-31 11-24-19](https://user-images.githubusercontent.com/97834160/229081538-893a946a-cbbe-4755-9ecf-eb72a1c6150b.png)
  - Figma:
![Screenshot from 2023-03-31 11-25-48](https://user-images.githubusercontent.com/97834160/229081735-42bdfbce-635c-4ae7-b104-3d0669c4e14a.png)

  - [ ] Kindly make sure your `see project` button stick to the Figma design:
  - Your design:
![Screenshot from 2023-03-31 11-26-19](https://user-images.githubusercontent.com/97834160/229082084-af8f459b-a6d5-4ba8-a081-ccfae721ae97.png)
 - Figma:
![Screenshot from 2023-03-31 11-27-36](https://user-images.githubusercontent.com/97834160/229082190-90f37216-56a2-4cd0-8294-9e6d6177ef84.png)

 - [ ] There is a gap between cards and  a background of  #E5E5E5;
  - Your design:
![Screenshot from 2023-03-31 11-28-41](https://user-images.githubusercontent.com/97834160/229082716-d0591827-8aea-4837-8725-23da4a949982.png)

  - Figma:
![Screenshot from 2023-03-31 11-30-18](https://user-images.githubusercontent.com/97834160/229082851-5cb65e32-38d8-4cf4-b12f-187243f0377b.png)

- [ ] The Get my resume button should stick to the Figma design in terms of background color, paddings and width:

  - Your design:
![Screenshot from 2023-03-31 11-31-00](https://user-images.githubusercontent.com/97834160/229083298-8e87e931-7e1e-4afb-b22f-073cfc0d94a0.png)

  - Figma:
![Screenshot from 2023-03-31 11-32-37](https://user-images.githubusercontent.com/97834160/229083462-98c486d4-ede6-4c11-962b-0b5c72df14ec.png)
 - [ ] Your skills list should also stick to the Figma design in terms of paddings and margins, kindly also include the > icons on the right edge.
   - Your design:
![Screenshot from 2023-03-31 11-36-27](https://user-images.githubusercontent.com/97834160/229084354-d3e3ee8b-09fc-4857-9846-b701770449b4.png)

   - Figma:
![Screenshot from 2023-03-31 11-33-18](https://user-images.githubusercontent.com/97834160/229084254-51cb4785-f00e-45de-a168-d53cacf13e2e.png)